### PR TITLE
Adjust Indentation in XAML Rules and Common props/targets/tasks

### PR DESCRIPTION
**Pull request by Nirmal4G**
Part of #4779

### Context

Make Common props, targets and tasks easier to read and understand.
Make Visual Studio's Property Page (_XAML_) Rules easier to read and understand.

Part of the larger refactoring that'll lead into #1686

### Changes Made

 - Remove all trailing spaces
 - Fixup new lines where necessary
   - Add new lines between every block to make it clear.
   - Remove unnecessary new lines to reduce scrolling.
 - Adjust Indentation as per EditorConfig (**2-space**)

### Notes

I'll also make sure not to mess up the git blame too much.
To make reviewing easier, I had split up the changes logically rather than a file.

**The commits by themselves are logical changes. So, if possible, rebase merge them to preserve file history view in Git.**

Also, please do note that these changes won't make much difference alone but with other formatting and refactors that follow this will make a world of difference in reading these files.

### CODE REVIEW

**Comment by Nirmal4G** [comment]
As you can see, some source files have also been touched. They belong to a different set of patches but I have extracted whitespace changes early on.

**Comment by Nirmal4G** [comment]
The PR #7165 removes the logic related to compatibility with MSBuild v4 and Dev11/Dev12. If that is approved, much of the changes here wouldn't apply. So, please wait until there's a verdict on that PR! And I'll rebase the PR accordingly.

File: src/Tasks/XamlRules/CSharp.BrowseObject.xaml [hunk 1]
```diff
@@ -1,4 +1,3 @@
-﻿<?xml version="1.0" encoding="utf-8"?>
```
Comment (Forgind) [inline src/Tasks/XamlRules/CSharp.BrowseObject.xaml:1]: Hmm...wondering if we should also update the copyright header in these files.

**Review by Forgind — APPROVED** [review]
Looks good!

File: src/Tasks/XamlRules/CSharp.BrowseObject.xaml (hunk 1 above)
Comment (Nirmal4G) [reply src/Tasks/XamlRules/CSharp.BrowseObject.xaml:1]: That's my next PR! 😏

**Comment by Nirmal4G** [comment]
@Forgind All done. Updated in response to #7165!

**Comment by Nirmal4G** [comment]
All done. Updated in response to #7165!

**Review by Forgind — APPROVED** [review]

**Comment by Nirmal4G** [comment]
Updated in response to #7169!

**Comment by Nirmal4G** [comment]
@Forgind This is finished BTW. And I can still push here but others not reflecting latest.

**Comment by BenVillalobos** [comment]
@Nirmal4G were these changes made via VS or some tool? The diff is massive, but if I can review each commit knowing it was done by automation it'd be easier to glance through instead of parsing every change.

Side note: this should merge as 1 commit

**Comment by Nirmal4G** [comment]
@BenVillalobos Only 2nd commit (_not all lines_) were done by me, manually.

> this should merge as 1 commit

You'll lose the clean diff, if not rebased. Anyway it's just my suggestion.

<details>
<summary>About my PR tree format</summary>


_I don't know what @Microsoft's policy is but where I work, we have archive policy with OSS projects, where we take all the contributions up until major or minor releases and clean them up. After a few major versions, we take that archive's latest as base. And we'll do this over and over again._

Hence, having separated like this will make our archiveists easier to cleanup the commit tree. It'll also make it easier to blame and digup historical commits easily and compare. Recently we also started using blame ignores to filter out the cosmetic commits.

</details>

**Comment by Nirmal4G** [comment]
@BenVillalobos Rebased and fixed merge conflicts

**Review by Forgind — APPROVED** [review]

**Review by BenVillalobos — APPROVED** [review]

**Pull request merged**